app/src/main/res/values/strings.xml

```diff
@@ -991,4 +991,27 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_drinking_water_potable_unsigned">Safe to drink (unsigned)</string>
     <string name="quest_drinking_water_not_potable_signed">Not safe to drink (signed)</string>
     <string name="quest_drinking_water_not_potable_unsigned">Not safe to drink (unsigned)</string>
+    <string name="quest_barrier_type_title">What type of barrier is this?</string>
+    <string name="quest_barrier_type_gate">Gate</string>
+    <string name="quest_barrier_type_lift_gate">Boom barrier</string>
+    <string name="quest_barrier_type_bollard">Bollard(s)</string>
+    <string name="quest_barrier_type_chain">Chain</string>
+    <string name="quest_barrier_type_rope">Rope</string>
+    <string name="quest_barrier_type_wire_gate">Removable wire section</string>
+    <string name="quest_barrier_type_cattle_grid">Grid blocking cattle from walking over</string>
```

Comment:
Is there any reason this doesn't just say "cattle grid" or is the idea to try and be more descriptive?

If so, it should probably be this (or some simplified word for livestock, animals, your dog wouldn't like it either?):
```suggestion
    <string name="quest_barrier_type_cattle_grid">Grid blocking livestock from walking over</string>
```

As it will stop sheep (and also horses) too.

Reply:
> Is there any reason this doesn't just say "cattle grid" or is the idea to try and be more descriptive?

At least I would have no idea what "cattle grid" is, or even that it applies to something road surface.

I tried similar explanation with kissing gate (failed) and did with wire gate/Hampshire gate.

Reply:
> > Is there any reason this doesn't just say "cattle grid" or is the idea to try and be more descriptive?
> 
> At least I would have no idea what "cattle grid" is, or even that it applies to something on a road surface (rather than vertical object).

Okay fair enough. I don't know if that's a language thing, or just down to the sort of environment/activies I was in growing up, or just being someone who walks a bit in the countryside.

Are you happy livestock isn't too "technical" then?

> I did similar explanation with wire gate/Hampshire gate. Also tried and failed with kissing gate.

Yeah while I might be able to summarise a kissing gate, I doubt I could get it down to six words, and I suspect certainly not without confusing some people who'd know what a kissing gate is, but possibly not a description (which I suspect would be angles and movement and stuff).